packages/cli/src/types/dataset.ts

```diff
@@ -0,0 +1,47 @@
+interface DatasetMetadata {
+  description: string;
+  url: string;
+  version: string;
+  year: number;
+  contributor: string;
+  date_created: string;
+}
+
+export interface ImageSchema {
```

Comment:
```suggestion
export interface DatasetImage {
```

Reply:
Sorry, just feel this looks more consistent than?

Reply:
resolved.